File: Java/wellmind-app/src/main/java/com/wellmind/dto/empresa/EmpresaDTO.java
package com.wellmind.dto.empresa;

import lombok.*;

import java.time.LocalDateTime;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class EmpresaDTO {

    private Long idEmpresa;
    private String nomeEmpresa;
    private String cnpj;
    private String cnpjFormatado;
    private String endereco;
    private String telefone;
    private String emailContato;
    private Boolean ativa;
    private LocalDateTime dataCadastro;
    private LocalDateTime ultimaAtualizacao;
    private Long totalColaboradores;
}

File: DotNet/WellMindApi/src/WellMindApi.Application/DTOs/Empresas/EmpresaDto.cs
namespace WellMindApi.Application.DTOs.Empresas;

/// <summary>
/// DTO de retorno de Empresa
/// </summary>
public record EmpresaDto
{
    public int IdEmpresa { get; init; }
    public string NomeEmpresa { get; init; } = string.Empty;
    public string CNPJ { get; init; } = string.Empty;
    public string CNPJFormatado { get; init; } = string.Empty;
    public string? Endereco { get; init; }
    public string? Telefone { get; init; }
    public string? EmailContato { get; init; }
    public DateTime DataCadastro { get; init; }
}
